resources/azure/testing/terraform/main.tf

```diff
@@ -116,7 +116,32 @@ resource "azurerm_key_vault_secret" "consumer_storage_key" {
   name         = "${azurerm_storage_account.consumer.name}-key1"
   value        = azurerm_storage_account.consumer.primary_access_key
   key_vault_id = azurerm_key_vault.main.id
-  depends_on = [
+  depends_on   = [
     azurerm_role_assignment.ci_key_vault
   ]
+}
+
+## CosmosDB account for integration testing. No need to create databases or containers,
+## they are created by the tests
+resource "azurerm_cosmosdb_account" "cosmosdb_integrationtest" {
+    name                     = "${var.prefix}-cosmosdb"
+  location            = "westeurope"
+  resource_group_name = azurerm_resource_group.main.name
+  offer_type          = "Standard"
+
+  enable_automatic_failover = false
+  enable_free_tier          = true
+
+  capabilities {
+    name = "EnableAggregationPipeline"
+  }
+
+  consistency_policy {
+    consistency_level = "Strong"
+  }
+
+  geo_location {
+    location          = "westeurope"
```

Comment:
same here as above,  azurerm_resource_group.main.location or var.location

Reply:
i had both of those hard-coded to "westeurope", because at the time I actually provisioned the resources, there was a service outage or overload in north europe (which is still the default)